framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1063,6 +1064,22 @@ public void postProcessClassTree(ClassTree tree) {
             // do so is here.
             wholeProgramInference.saveResults();
         }
+
+        if (tree.getExtendsClause() == null) {
```

Comment:
This test seems suspect to me.  I would think that an implicit extends clause can conflict with an explicit annotation on the class declaration.
If so, then there should be a test for that, and also logic to handle it.

Reply:
As we discussed yesterday, we assume that a class implicitly extends `@Top Object` and there is no need for any check in this case.